Pipfile

```diff
@@ -38,7 +38,7 @@ eventlet = "==0.33.1"
 flask = "~=2.1.2"
 flask-bcrypt = "==1.0.1"
 flask-marshmallow = "==0.14.0"
-flask-migrate = "==3.1.0"
+flask-migrate = "~=3.1.0"
```

Comment:
[suggestion] if `flask-migrate` does proper semantic versioning, it'd probably be even better to switch this to `~=3.1` to get all updates in the `3.x` series

(edit) lol, just looked at the version history, no need to change this. I do think, however, that in general use we'd be better off with the `~=x.y` format the next time we update anything

Reply:
Yeah, I changed this when I was having CLI problems...and then forgot about it. Maybe I'll go ahead and unpin it while we're here.